src/python/bot/minimizer/html_minimizer.py

```diff
@@ -190,12 +190,12 @@ def get_tokens_and_metadata(data):
   @staticmethod
   def combine_worker_tokens(tokens, prefix='', suffix=''):
     """Combine tokens for a worker minimizer."""
-    return '%s%s%s' % (prefix, ''.join(tokens), suffix)
+    return prefix + b''.join(tokens) + suffix
 
   @staticmethod
   def combine_tokens(tokens):
     """Combine tokens into a usable format, stripping metadata."""
-    return ''.join([str(t) for t in tokens])
+    return ''.join([bytes(t) for t in tokens])
```

Comment:
'' need to b'' ?

>>> '' + b'a' + ''
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate str (not "bytes") to str
>>> ''.join([b'a'])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: sequence item 0: expected str instance, bytes found

Reply:
Good catch.